docs/developing.rst

```diff
@@ -110,6 +110,7 @@ performing these checks locally avoids unnecessary build failures.
 
     # check code for linting errors
     flake8
+    mypy --no-incremental src/streamlink/ src/streamlink_cli/ tests/ docs/ setup.py
```

Comment:
```suggestion

    # check code for typing errors
    mypy
```

Reply:
Thanks. I committed the change just now, sorry for the late.